packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -69,7 +73,8 @@ const knowClassesToAnalyzeForImages = <WildcardMatch, List<String>>{
 
   // Anything that exactly matches:
   WildcardMatch.exact: [
-    '_Int32List',
+    '_Int32List', // Emulator stores raw images as Int32List.
+    '_Int64List', // New devices e.g., Pixel 3 XL stores images as Int64List.
```

Comment:
wild guess: it isn't emulators vs new devices, it is 32 bit vs 64bit devices.

Reply:
It's the device good guess bad comment.  Fixed.